mappings/net/minecraft/level/chunk/Chunk.mapping

```diff
@@ -1,14 +1,16 @@
 CLASS net/minecraft/class_43 net/minecraft/level/chunk/Chunk
 	FIELD field_954 tiles [B
 	FIELD field_956 level Lnet/minecraft/class_18;
-	FIELD field_957 data Lnet/minecraft/class_257;
+	FIELD field_957 metadata Lnet/minecraft/class_257;
```

Comment:
what is this field for. Storing item/tile metadata?

Reply:
tile meta

Reply:
ok yeah maybe just name it `meta`

Reply:
concise go brrr